<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ipAddressEdit">
      <property name="text">
       <string>192.168.20.104</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>RIGHT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>LEFT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>DOWN</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>239</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="presetGotoButton">
      <property name="text">
       <string>Goto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="presetDeleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="presetSetButton">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="3">
     <widget class="QRadioButton" name="p3_radioButton">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="p1_radioButton">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="p0_radioButton">
      <property name="text">
       <string>0</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="p2_radioButton">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QRadioButton" name="p15_radioButton">
      <property name="text">
       <string>15</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QRadioButton" name="p14_radioButton">
      <property name="text">
       <string>14</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="p13_radioButton">
      <property name="text">
       <string>13</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="p12_radioButton">
      <property name="text">
       <string>12</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QRadioButton" name="p11_radioButton">
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QRadioButton" name="p10_radioButton">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="p9_radioButton">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="p8_radioButton">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="p7_radioButton">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="p6_radioButton">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="p5_radioButton">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="p4_radioButton">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>241</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>169</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Flicker</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmosSetButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>169</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>186</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="colorAuto_radioButton">
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="colorColor_radioButton">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="colorBW_radioButton">
       <property name="text">
        <string>B/W</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>172</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="flicker50_radioButton">
       <property name="text">
        <string>50Hz</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="flicker60_radioButton">
       <property name="text">
        <string>60Hz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>169</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>IR Cut</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>171</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Mirror Settings:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QRadioButton" name="mirrorLR_radioButton">
         <property name="text">
          <string>Left /  Right</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="mirrorUD_radioButton">
         <property name="text">
          <string>Up / Down</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="mirrorUDLR_radioButton">
         <property name="text">
          <string>U/D L/R</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="mirrorNormal_radioButton">
         <property name="text">
          <string>Normal</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>174</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Exposure Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="exposureTime_comboBox">
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/60</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/125</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/250</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/2000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/5000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/10000</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>299</y>
      <width>172</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sensitivity</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="irCutSetting_comboBox">
       <item>
        <property name="text">
         <string>Daytime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Night</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ai_Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IR_Auto</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="irCutSensitvity_spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>White Balance</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="wbIndoor_radioButton">
         <property name="text">
          <string>Indoor</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="wbOutdoor_radioButton">
         <property name="text">
          <string>Outdoor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>169</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mirrorNormal_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mirrorUD_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mirrorLR_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mirrorUDLR_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wbIndoor_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wbOutdoor_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flicker50_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flicker60_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureTime_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>irCutSetting_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>irCutSensitvity_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorAuto_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorColor_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorBW_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>cmosSetButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
